Rakefile

```diff
@@ -23,7 +23,9 @@ end
 
 desc 'Compile the SVG icons into the font'
 task :compile do
-  system 'fontcustom compile'
+  result = `fontcustom compile`
+  puts result
+  raise StandardError if result =~ /error/
```

Comment:
The problem is that fontcustom almost always exits with a `0` status :disappointed: